.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/SOILWAT2"]
 	path = src/SOILWAT2
 	url = https://github.com/DrylandEcology/SOILWAT2
-	branch = release/devel_v7.0.0
+	branch = read_more_daily_input
```

Comment:
Running `data-raw/prepare_testInput_objects.R` updates output objects? Why is that? Shouldn't this produce exactly the same output given that all new daily inputs are turned off by default?